Client/Ecosia/UI/TreeCounterCell.swift

```diff
@@ -76,10 +76,7 @@ final class TreeCounterCell: UICollectionViewCell, Themeable {
                 counter.text = formatter.string(from: .init(value: count))
             })
         }
-
-        treeCounter.update(session: .shared) {
-            UserDefaults.statistics = try? JSONEncoder().encode($0)
```

Comment:
this was written but never read. We used this in the old app for bringing the TreeCount to extensions, but we don't need this anymore